### PR TITLE
Sql checker having

### LINKED ISSUE #660: Extension of the SQL Checker (Having)
The SQL Checker can already analyze a large number of MySQL queries. Currently, it is not yet possible to recognize "Having". The Checker must be extended accordingly to enable detection. The parser (Mo_SQL_Parsing) recognizes "having", but the respective parameter from the JSON must be analyzed.

### PR DESCRIPTION
**Pull request by Artorin**
resolves #660

### CODE REVIEW

**Comment by Jonas960** [comment]
@Artorin I found a problem with queries like this: 
"SELECT Name, AVG( Quantity ) FROM Products GROUP BY Name HAVING AVG( Quantity ) > 3* (SELECT AVG( Quantity ) FROM SalesOrderItems);" 
The function just store the following parameter:
image

Please think about a solution for this.

**Review by Jonas960 — CHANGES_REQUESTED** [review]
@Artorin I found a problem with queries like this:
"SELECT Name, AVG( Quantity ) FROM Products GROUP BY Name HAVING AVG( Quantity ) > 3* (SELECT AVG( Quantity ) FROM SalesOrderItems);"
The function just store the following parameter:

image


Please think about a solution for this.

**Comment by Artorin** [comment]
testen

**Review by Jonas960 — DISMISSED** [review]
Test was performed with different queries. No errors occurred.

File: modules/fbs-core/web/build.gradle [hunk 1]
```diff
@@ -37,7 +37,7 @@ distributions {
 // It will work on evry system, because of relative paths (at leat I hope so)
 
 task copyWebToWS(type: Copy, dependsOn: [build]) {
-    from "${projectDir}/dist/web" // path from distributed web-gui
+    from "${projectDir}/dist/${project.name}" // path from distributed web-gui
```
Comment (mxsph) [inline modules/fbs-core/web/build.gradle:40]: Please reverse this change as it will break the build process of the frontend.

**Review by mxsph — CHANGES_REQUESTED** [review]

File: modules/fbs-core/web/build.gradle (hunk 1 above)
Comment (Jonas960) [reply modules/fbs-core/web/build.gradle]: done

**Review by Jonas960 — APPROVED** [review]
SQL part of the PR is still ok!

**Review by Medienkunzt — APPROVED** [review]

**Review by mxsph — APPROVED** [review]

**Pull request merged**